app/views/case_studies/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= javascript_pack_tag "case_study_page" %>
+<%= stylesheet_pack_tag "case_study_page" %>
```

Comment:
I don't think you need to add stylesheet here. It works only for base css components like buttons or pagination...